scala/scala_maven_import_external.bzl

```diff
@@ -307,3 +311,28 @@ scala_maven_import_external = repository_rule(
         "default_visibility": attr.string_list(default=["//visibility:public"]),
         "extra_build_file_content": attr.string(),
     })
+
+def scala_maven_import_external(artifact, server_urls, **kwargs):
+  jvm_maven_import_external(
+    rule_name = "scala_import",
+    rule_load = "load(\"@io_bazel_rules_scala//scala:scala_import.bzl\", \"scala_import\")",
```

Comment:
this should probably be an attribute with a default value since some organizations (like us) add these in prelude and also some organizations (stripe?) might have aliases or stuff so they load from a different place

Reply:
done